lib/identity_cache/query_api.rb

```diff
@@ -108,7 +108,7 @@ def prefetch_associations(associations, records)
 
       def raise_if_scoped
         if current_scope
-          IdentityCache.logger.error("#{name} has scope: #{current_scope.where_values_hash.inspect}")
+          IdentityCache.logger.error("#{name} has scope: #{current_scope.to_sql} (#{current_scope.values.keys})")
```

Comment:
Most of the information we will want is in the SQL. Dylan asked for the keys because it will show things like preloading.